src/sound.ts

```diff
@@ -1,9 +1,9 @@
+
 /**
  * @copyright freeCodeCamp
  * @license BSD-3
  * @author Naomi Carrigan
  */
-
```

Comment:
This formatting change should be reverted

Reply:
No problem artichoke.